### PR TITLE
Release 2.6.0

### PR DESCRIPTION
**Pull request by jonasscheid**
### `Added`

- Added MS²Rescore module with the underlying python CLI [#293](https://github.com/nf-core/mhcquant/pull/293) 🤖  
- Added support for handling various archive formats: `d|d.tar.gz|d.tar|d.zip|mzML.gz|raw|RAW|mzML` [#323](https://github.com/nf-core/mhcquant/pull/323) 
--> Provide PRIDE ftp link of raw data in samplesheet to directly stage + process projects 🥳 

### `Fixed`

- Create only one decoy database [#299](https://github.com/nf-core/mhcquant/pull/299) 🚀 
- Use `groupKey` to streamline group-wise processing [#310](https://github.com/nf-core/mhcquant/pull/310) 🚀 
- Replaced local modules with nf-core modules [#310](https://github.com/nf-core/mhcquant/pull/310) 🍏 

### `Comments`
- `--use_deeplc` and `--use_ms2pip` were removed and can now be specified via `--feature_generators deeplc,ms2pip`
- `--ms2pip_model_name` was renamed to `--ms2pip_model`

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/mhcquant/tree/master/.github/CONTRIBUTING.md)
- [x] If necessary, also make a PR on the nf-core/mhcquant _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: docs/usage.md
Comment (marissaDubbelaar) [inline docs/usage.md:80]: Instead of search parameters, maybe use some parameters for a class I search that we use most often?

File: docs/usage.md
Comment (marissaDubbelaar) [inline docs/usage.md:82]: Then maybe also alter this to 12 or 15, unless we use 14 now 😅

File: workflows/mhcquant.nf
Comment (marissaDubbelaar) [inline workflows/mhcquant.nf:120]: This if-and-else statement can also be put into the module.config right?

File: workflows/mhcquant.nf
Comment (marissaDubbelaar) [inline workflows/mhcquant.nf:145]: Same goes for all of the if-else-statements

**Review by marissaDubbelaar — APPROVED** [review]
A lot of effort went into this PR!
On a small note, I would change the version to 2.0 since many changes were made, and the prediction of classes and rt is retired at the moment.
Maybe someone from the core team or contributors can give a second review.

File: docs/usage.md
Comment (jonasscheid) [reply docs/usage.md:82]: Use 14 currently since thats the max length to predict on with netmhcpan 😁

File: workflows/mhcquant.nf
Comment (jonasscheid) [reply workflows/mhcquant.nf:120]: I don't think so since I need to call a different combination of modules in the workflow depending on that parameter 🤔

File: workflows/mhcquant.nf
Comment (jonasscheid) [reply workflows/mhcquant.nf:145]: Same here I guess..

File: docs/output.md
Comment (mirpedrol) [inline docs/output.md:29]:
```suggestion
<summary>TSV Quant</summary>
```

File: modules/local/ms2rescore.nf
Comment (mirpedrol) [inline modules/local/ms2rescore.nf:11]: would it be better to add this to the `modules.config` file? that way it can be overridden if necessary

File: modules/local/ms2rescore.nf
Comment (mirpedrol) [inline modules/local/ms2rescore.nf:1]: could this be made an nf-core/module?

Comment (jonasscheid) [reply modules/local/ms2rescore.nf:1]: I don't think so, since we need to use the python API to get the desired output of the tool. In general you can call this tool from the command line and create an nf-core module out of it, but then we could not use it.

File: modules/local/ms2rescore.nf
Comment (jonasscheid) [reply modules/local/ms2rescore.nf:11]: I think it ended up here because `workflow.containerEngine` is not accessible in `conf/modules.config`.

**Comment by jonasscheid** [comment]
Thanks for the reviews!

**Pull request merged**
